.github/workflows/tests.yaml

```diff
@@ -14,12 +20,19 @@ jobs:
         cache: 'pip' # caching pip dependencies
     - run: pip install -r requirements.txt
     - name: Run unittests
-      run: python -m unittest -v
+      run: pytest --cov --cov-report=term-missing --cov=weaviate --cov-report xml:coverage.xml test
+    - name: Upload to codecov
```

Comment:
Does this publish the results somewhere? Do we need an account for this?

Reply:
We don't need an account, the results are added [as a reply to the PR](https://github.com/semi-technologies/weaviate-python-client/pull/137#issuecomment-1281898094).
(not the biggest fan of this, but its how it's done in weaviate-core and I just copied everything over)

Reply:
Nice. Thanks for the info.